client/ayon_core/tools/sceneinventory/models/containers.py

```diff
@@ -219,72 +226,79 @@ def get_container_items_by_id(self, item_ids):
             for item_id in item_ids
         }
 
-    def get_representation_info_items(self, representation_ids):
+    def get_representation_info_items(self, project_name, representation_ids):
         output = {}
-        missing_repre_ids = set()
+        missing_repre_ids_by_project = {}
```

Comment:
Funnily enough - this now doesn't need to be "per project" anymore - because the full method operates within the project's context. (by having a `project_name` input)

Reply:
> maybe we can do this in separate PR as enhancement.

We should implement the support correctly. It should not be that hard, it requires a lot of changes in code, but in practice it means we just have to consider project name as one hierarchy level above everything else (everywhere). If you struggle to achieve that, let me know.